{{cookiecutter.project_slug}}/src/{{cookiecutter.project_module}}/app.py

```diff
@@ -49,12 +43,19 @@ def init_app(application, interface):
 
     # Add routes and resources.
     from {{cookiecutter.project_module}} import resources
-    interface.add_resource(resources.HelloWorld, "/")
-    interface.init_app(application)
+    resources.init_app(application)
 
     # Add CORS information for all resources.
     CORS(application)
 
+    # Add an error handler for webargs parser error, ensuring a JSON response
+    # including all error messages produced from the parser.
+    @app.errorhandler(422)
+    def handle_webargs_error(error):
+        response = jsonify(error.data['messages'])
```

Comment:
Shouldn't we use an error schema here instead?

Reply:
We could marshal all resource endpoints with a schema for HTTP 422. You cannot marshal an error handler, so the benefit would be documentation.